package.json

```diff
@@ -10,6 +10,7 @@
   },
   "devDependencies": {
     "eslint": "^9.0.0",
+    "@stylistic/eslint-plugin": "^3.0.0",
```

Comment:
This includes typescript. Javascript-only version is `@stylistic/eslint-plugin-js`.

Reply:
I have no idea how my old experiments with that did not worked. But now it does without ts dependency.